packages/perseus/src/widgets/label-image.tsx

```diff
@@ -503,17 +512,17 @@ export class LabelImage extends React.Component<
 
         const {activeMarkerIndex, markersInteracted} = this.state;
 
-        // Render all markers for widget.
-        return markers.map((marker, index): React.ReactElement => {
-            // Determine whether page is rendered in a narrow browser window.
-            const isNarrowPage = window.matchMedia(
-                mediaQueries.xsOrSmaller.replace("@media ", ""),
-            ).matches;
+        // Determine whether page is rendered in a narrow browser window.
+        const isNarrowPage =
+            this._mounted &&
+            window.matchMedia(mediaQueries.xsOrSmaller.replace("@media ", ""))
+                .matches;
```

Comment:
**note:** Moved out of the loop since it won't change for each item, and modified to return `false` on initial render so that it will SSR and hydrate correctly.